### PR TITLE
SISRP-26666 - Advising Resources - should not include 'Schedule of Classes - Scheduler View' link

### PR DESCRIPTION
**Pull request by dfkaye**
https://jira.berkeley.edu/browse/SISRP-26666

+ removes the Schedule of Classes link from Dashboard Advisor Resources
+ adds hover text to WebNow Documents link in Dashboard Advisor Resources
+ adds hover text to student-specific Schedule Planner link on Student Overview (Advisor Resources)

Release Notes for 7.1 updated - remove `cs_advising_scheduler_view` flag from all environments:
https://confluence.ets.berkeley.edu/confluence/pages/viewpage.action?spaceKey=MYB&title=Release-Specific+Instructions

### CODE REVIEW

File: app/models/campus_solutions/advising_resources.rb [hunk 1]
```diff
@@ -37,8 +37,8 @@ def build_feed(response)
 
       # The following links are hard-coded, for now. Ideally they would be served by CS API but there is an urgent need
       # thus we manage the content via CalCentral settings.
-      add_cs_link links, :web_now_documents, 'WEB_NOW_DOCUMENTS', 'WebNow Documents'
-      add_cs_link links, :schedule_planner, 'SCHEDULE_PLANNER_STUDENT_SPECIFIC', 'Schedule Planner', "?EMPLID=#{student_empl_id}"
+      add_cs_link links, :web_now_documents, 'WEB_NOW_DOCUMENTS', 'WebNow Documents', '', 'View uploaded student documents in the Image Now dashboard.'
+      add_cs_link links, :schedule_planner, 'SCHEDULE_PLANNER_STUDENT_SPECIFIC', 'Schedule Planner', "?EMPLID=#{student_empl_id}", "View this student's planned schedule for this term."
```
Comment (lyttam) [inline app/models/campus_solutions/advising_resources.rb]: Just curious, why does the hover text have to be hard-coded?  Is the CS API not sending it to us yet?

Comment (dfkaye) [inline app/models/campus_solutions/advising_resources.rb]: @lyttam - yes, these links come from another campus solutions api so we have to hard-code the hover text - better here in the model than in the template.

File: src/assets/templates/widgets/toolbox/user_preview.html [hunk 2]
```diff
@@ -108,9 +108,10 @@ <h3 data-ng-bind="targetUser.error.summary"></h3>
                   ></div>
                 </div>
                 <div data-ng-if="ucAdvisingResources.links.schedulePlannerStudentSpecific.url">
-                  <a
-                    data-ng-href="{{ucAdvisingResources.links.schedulePlannerStudentSpecific.url}}"
-                    data-ng-bind="ucAdvisingResources.links.schedulePlannerStudentSpecific.name"></a>
+                  <div
+                    data-cc-campus-solutions-link-item-directive
+                    data-link="ucAdvisingResources.links.schedulePlannerStudentSpecific"
+                  ></div>
```
Comment (davidlimcheng) [inline src/assets/templates/widgets/toolbox/user_preview.html]: @dfkaye Is this done to add hover-text for the Schedule Planner link?  If so, don't we need to add an `data-ng-title` (or something similar) to the WebNow Documents `<div>` below, on line 122?

Comment (dfkaye) [inline src/assets/templates/widgets/toolbox/user_preview.html]: @davidlimcheng - not anymore :) I modified the link directive a while back to look for a `title` property and create a title attribute on the new link it's there.

Comment (davidlimcheng) [inline src/assets/templates/widgets/toolbox/user_preview.html]: @dfkaye Ahh I see, and I do see now that the WebNow `<div>` has a `data-ng-attr-title` attribute.  Good stuff!

Comment (dfkaye) [inline src/assets/templates/widgets/toolbox/user_preview.html]: @davidlimcheng - per the QA PR I'll remove the `cs_advising_scheduler_view` from the settings.yml files and set release instructions to remove them from the upstream environments

**Comment by johncrossman** [comment]
Thanks @dfkaye  Looks good.

**Comment by dfkaye** [comment]
thanks @johncrossman :)

**Pull request merged**